docs/logs/faq.md

```diff
@@ -442,6 +442,8 @@ FISMA, DIACAP, FedRAMP, and HIPAA.  For more information about AWS
 security and compliance certifications see:
 <http://aws.amazon.com/security> and <https://aws.amazon.com/compliance/>.
 
+Your data is protected from unintentional reads. If you want to read data using scripts you acn authorize your API calls using `apiKey`. You can read more about this [here](search-through-the-elasticsearch-api) and [here](/apiKey).
```

Comment:
Reads might be intentional :).  They may just be..... undesirable?  Unauthorized?
acn ==> can